packages/ckeditor5-table/theme/tablelayout.css

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-licensing-options
+ */
+
+.ck-editor__editable {
+	&.ck-editor__editable_inline {
+		& > .ck-widget.ck-widget_with-selection-handle.layout-table:first-child {
+			margin-top: var(--ck-spacing-large);
+		}
+	}
+
+	& .table.layout-table {
+		/* Do not reserve space above and below the layout table. */
+		margin-top: 0;
+		margin-bottom: 0;
+
+		/* Widget type around overrides. */
+		&.ck-widget {
+			&:hover > .ck-widget__selection-handle {
+				opacity: 0.75;
+				visibility: visible;
+
+				&:hover {
+					opacity: 1;
+				}
+			}
+
+			&:has( .ck-widget:hover ) > .ck-widget__selection-handle {
+				opacity: 0;
+				visibility: hidden;
+			}
```

Comment:
Is this needed? Looks like it is already in widget.css.

Reply:
Yes, it is. I've checked and  because of setting the opacity to 0.75 (Line 21) it overrides the similar selector in widget (specificity is higher), thats why we must override it here also.